pkgs/applications/misc/gcfflasher/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, pkgs
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gcfflasher";
+  version = "4.0.3-beta";
+
+  src = fetchFromGitHub {
+    owner = "dresden-elektronik";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-m+iDBfsHo+PLYd3K8JaKwhIXcnj+Q8w7gIgmHp+0plk=";
+  };
+
+  nativeBuildInputs = with pkgs; [ gcc pkg-config libgpiod ];
+
+  buildPhase = ''
+    ./build_posix.sh
+  '';
```

```suggestion
  buildPhase = ''
    ./build_posix.sh
  '';
```

If you look at the repo root, you can see they have ` CMakeLists.txt`, which indicates cmake usage. Add `cmake` to `nativeBuildInputs` and its hooks should take care of the build.